.github/workflows/linkcheck.yml

```diff
@@ -0,0 +1,16 @@
+name: Linkspector
+on: [pull_request]
+jobs:
+  check-links:
+    if: ${{ github.repository != 'stfc/PSyclone-mirror' }}
+    name: runner / linkspector
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run linkspector
+        uses: umbrelladocs/action-linkspector@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
```

Comment:
I think I like more the "github-pr-check" reports inlined in the code than the comments (which look quite noisy), but this is subjective. What do you think?

Reply:
Yes, I'm happy to change it.